euth/comments/models.py

```diff
@@ -2,6 +2,7 @@
 from django.contrib.contenttypes.fields import GenericForeignKey
 from django.contrib.contenttypes.models import ContentType
 from django.db import models
+from django.utils.translation import ugettext as _
```

Comment:
Models should always use `ugettext_lazy`, see [Django documentation: Lazy translation](https://docs.djangoproject.com/en/1.10/topics/i18n/translation/#lazy-translations).